### ISSUE TITLE
Page Lag

### ISSUE THREAD

**Issue by SaySo02** [lllyasviel/stable-diffusion-webui-forge#1922]
I updated Forge to the last possible update i guess, and i seen that everything just work as fine as always, what happens? the UI lags, and alot, at the point that it has to wait like 7 seconds to appear the autocomplete tag from danbooru, any way to reduce the lag from the stable diffusion page? [FORGE]

**Comment by wzwow** [comment]
yes prompt box become lag.... text only appears after 5 sec of typing :(

**Comment by Hunt3rseeker** [comment]
I have this issue too. The prompt box is quite laggy; a slight, but visible delay when I'm typing, and a greater delay until autocomplete catches on.

**Comment by supersonic13** [comment]
I've seen this before, and think this is almost certainly due to extensions which are not fully compatible with the new forge/Gradio 4. 
Try disabling any 3rd part extensions and see if the problem disappears ?
